package.json

```diff
@@ -96,5 +108,10 @@
     "stepDefinitions": [
       "tests/e2e/support/step-definitions/**/*.{js,ts}"
     ]
+  },
+  "eslintConfig": {
```

Comment:
It's better to move eslint plugin for storybook to eslint.config file

Reply:
Thanks! Fixed